webapi/Skills/ChatSkills/ExternalInformationSkill.cs

```diff
@@ -153,25 +159,20 @@ public class ExternalInformationSkill
             Plan? plan = null;
             // Use default planner options if planner options are null.
             var plannerOptions = this._planner.PlannerOptions ?? new PlannerOptions();
-            int retriesAvail = plannerOptions.MissingFunctionError.AllowRetries
-                ? plannerOptions.MissingFunctionError.MaxRetriesAllowed // Will always be at least 1
-                : plannerOptions.AllowRetriesOnInvalidPlan ? 1 : 0;
+            int retriesAvail = plannerOptions.PlannerError.AllowRetries
```

Comment:
Is this correct??

When we allow retries, retriesAvail is set to 0, otherwise it's MaxRetriesAllowed???

It feels like it should be the other way around.

If this is indeed correct, please put a comment to explain this logic

Reply:
Good catch!